server/src/main/java/org/apache/uniffle/server/buffer/ShuffleBufferManager.java

```diff
@@ -712,4 +712,8 @@ public boolean limitHugePartition(
     }
     return false;
   }
+
+  public void setUsedMemory(long usedMemory) {
```

Comment:
This function should only be used for testing purposes ?

Reply:
Yes.